pkg/data/dashboard/namespace.go

```diff
@@ -11,14 +12,26 @@ import (
 )
 
 func addCattleGlobalNamespaces(ctx context.Context, k8s kubernetes.Interface) error {
+	if features.Fleet.Enabled() {
+		_, err := k8s.CoreV1().Namespaces().Get(ctx, "fleet-local", metav1.GetOptions{})
+		if apierrors.IsNotFound(err) {
+			_, err = k8s.CoreV1().Namespaces().Create(ctx, &corev1.Namespace{
```

Comment:
err not handled here?

Reply:
This is fine because it's reset and returned. We know the original error is not found, so we reassign and return that

Reply:
oh it is assigned to err in the next if